pom.xml

```diff
@@ -31,17 +31,18 @@
         <maven.build.timestamp.format>yyyyMMdd</maven.build.timestamp.format>
         <timestamp>${maven.build.timestamp}</timestamp>
 
-        <hazelcast.version>4.1</hazelcast.version>
+        <hazelcast.version>5.0</hazelcast.version>
 
-        <log4j.version>1.2.12</log4j.version>
+        <log4j.version>2.23.1</log4j.version>
         <junit.version>4.13.1</junit.version>
 
         <jsp.api.version>2.2.1</jsp.api.version>
         <servlet.api.version>3.1.0</servlet.api.version>
-        <org.springframework.version>4.3.9.RELEASE</org.springframework.version>
-        <spring-security.version>4.2.3.RELEASE</spring-security.version>
-        <jetty.maven.plugin.version>8.1.2.v20120308</jetty.maven.plugin.version>
+        <org.springframework.version>5.3.34</org.springframework.version>
+        <spring-security.version>5.8.12</spring-security.version>
+        <jetty.maven.plugin.version>9.4.54.v20240208</jetty.maven.plugin.version>
```

Comment:
We need to figure out what versions of Jetty and Tomcat we should support with this new version. This should be based on what is maintained on Jetty / Tomcat side and what our customers currently use.

Reply:
This is just a test dependency, you mean for better testing (better means close to their environment) right ?